composer.json

```diff
@@ -71,7 +73,7 @@
   "extra": {
     "branch-alias": {
       "dev-master": "1.0-dev",
-      "dev-develop": "1.1-dev"
+      "dev-prooph_v7": "0.3-dev"
```

Comment:
As [v0.3 has been released](https://github.com/prooph/service-bus-symfony-bundle/releases/tag/v0.3) this should be `0.4-dev`